Examples/Algorithms/Geant4HepMC/src/EventRecording.cpp

```diff
@@ -16,6 +16,8 @@
 #include <stdexcept>
 
 #include <FTFP_BERT.hh>
+#include <G4RunManager.hh>
+#include <G4VUserDetectorConstruction.hh>
```

Comment:
Isn't this already included from the header?

Reply:
It's probably not strictly necessary, but doesn't hurt to have. I think I added this for the python bindings, and didn't remove for this PR. 

I can strip these if you prefer.

Reply:
I fine with both and I don't want to block this trivial PR by this detail.